graylog2-server/src/main/java/org/graylog2/plugin/PluginModule.java

```diff
@@ -323,4 +324,13 @@ protected void addAuthServiceBackend(String name,
         authServiceBackendBinder().addBinding(name).to(factoryClass);
         registerJacksonSubtype(configClass, name);
     }
+
+    protected MapBinder<String, PluginUISettingsProvider> pluginUISettingsProviderBinder() {
+        return MapBinder.newMapBinder(binder(), String.class, PluginUISettingsProvider.class);
```

Comment:
We need to make sure the map binder exists even when there is no plugin that's calling `addPluginUISettingsProvider()`. Otherwise the server will not start.

Similar to the following. We can initialize the settings map binder there as well.

https://github.com/Graylog2/graylog2-server/blob/e8dd88cf71296c13c30b5ba4de6f559ca9500a21/graylog2-server/src/main/java/org/graylog2/shared/bindings/PluginBindings.java#L42-L46

Reply:
Will add.